### PR TITLE
docs: makes glee internal features into pages

### PR DESCRIPTION
**Pull request by afzal442**
**Description**
This PR recreates glee internal features into pages

**Related issue(s)**
* Related to #431 
* Part of #480

### CODE REVIEW

**Review by Souvikns — CHANGES_REQUESTED** [review]
Should these docs go under the pages folder just like the [installation doc](https://github.com/asyncapi/glee/blob/master/docs/pages/installation.md) also is the pages folder for rending on the website?

**Comment by afzal442** [comment]
Hey @Souvikns sorry for the delay

> Should these docs go under the pages folder ...
I think let's first render the existing pages like installation guide and see how it looks

> is the pages folder for rending on the website?
Yes using github action as described here https://github.com/asyncapi/glee/issues/481

File: docs/app-structure.md
Comment (KhudaDad414) [inline docs/app-structure.md:1]: @afzal442 why are we removing this file? 🤔

Comment (afzal442) [reply docs/app-structure.md:1]: Sorry @KhudaDad414! not removing. but recreating it into pages.

**Review by KhudaDad414 — APPROVED** [review]

**Comment by KhudaDad414** [comment]
Let's wait for @Souvikns review.

**Comment by afzal442** [comment]
pinging @Souvikns 😃

**Review by Souvikns — APPROVED** [review]

**Pull request merged**